hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/compact/TestHoodieCompactor.java

```diff
@@ -129,6 +152,10 @@ public void testCompactionEmpty() {
       String compactionInstantTime = HoodieActiveTimeline.createNewInstantTime();
       Option<HoodieCompactionPlan> plan = table.scheduleCompaction(context, compactionInstantTime, Option.empty());
       assertFalse(plan.isPresent(), "If there is nothing to compact, result will be empty");
+
+      // Verify compaction.requested, compaction.completed metrics counts.
+      assertEquals(0, getCompactionMetricCount("counter", "compaction.requested"));
```

Comment:
can we re-use the variables here instead of hard coding the metric names ?

Reply:
I can use variables, but from what I could see referring to metric names using inline literal strings appears to be the common pattern in the codebase and hence this usage. Let me know what you think.

Reply:
@amrishlal, you could use  const variable `REQUESTED_COMPACTION_EXTENSION` in `HoodieTimeline`.

Reply:
@SteNicholas Using `HoodieTimeline.REQUESTED_COMPACTION_SUFFIX` and `HoodieTimeline.COMPLETED_COMPACTION_SUFFIX` as `REQUESTED_COMPACTION_EXTENSION` was leading to a double dot in the final metric name.